test/unit/helpers/MeasureBundleHelpers.test.ts

```diff
@@ -788,10 +789,35 @@ describe('MeasureBundleHelpers tests', () => {
     });
   });
 
-  describe('extractLibrariesFromBundle', () => {
+  describe('extractLibrariesFromLibraryBundle', () => {
+    it('properly gets libraries from EXM130 library bundle', () => {
+      const measureBundle = getJSONFixture('EXM130-7.3.000-bundle-nocodes.json') as fhir4.Bundle;
+      const libraryBundle: fhir4.Bundle = {
+        resourceType: 'Bundle',
+        id: 'EXM130-7.3.000-bundle',
+        type: 'transaction'
+      };
+      libraryBundle.entry = measureBundle.entry?.filter(e => e.resource?.resourceType === 'Library');
+
+      const rootLibRef = 'Library/library-EXM130-7.3.000';
```

Comment:
We should test root library extraction using canonical URLs as well, not just logical references.

The logical reference stuff is actually legacy support for how some old connectathon measures were defined. Using "url" as the root library reference is preferable, and conformant to the latest FHIR spec